exporting/mongodb/README.md

```diff
@@ -0,0 +1,28 @@
+# MongoDB exporting connector
+
+## Prerequisites
+
+To use MongoDB as an external storage for long-term archiving, you should first
+[install](http://mongoc.org/libmongoc/current/installing.html) `libmongoc` 1.7.0 or higher. Next, re-install Netdata
+from the source, which detects that the required library is now available.
+
+## Configuration
+
+To enable data exporting to a MongoDB database, run `./edit-config exporting.conf`
+in the Netdata configuration directory and set the following options:
+
+```conf
+[mongodb:my_instance]
+    enabled = yes
+    destination = mongodb://<hostname>
+    database = your_database_name
+    collection = your_collection_name
+```
+
+You can find more information about the `destination` string URI format in the MongoDB
+[documentation]](https://docs.mongodb.com/manual/reference/connection-string/)
```

Comment:
I think we have a typo problem here, please, remove a `]` from `[documentation]]`.

Reply:
Fixed